lib/mumukit/auth/role.rb

```diff
@@ -17,15 +30,33 @@ def to_sym
       @symbol
     end
 
+    def broader_than?(other)
+      other.narrower_than? self
+    end
+
+    def narrower_than?(other)
+      other.class != self.class && narrower_than_other?(other)
+    end
+
+    def to_mumukit_role
+      self
+    end
+
     private
 
-    def self.parent(parent)
-      define_method(:parent) { self.class.parse(parent) }
+    def narrower_than_other?(other)
```

Comment:
Couldn't this recurse over `narrower_than_other?` instead of `narrower_than?`
though it's just a few less comparisons, not a big deal

Reply:
You are right. It has the drawback it can not be truly private, but I will change it.